src/menu.rs

```diff
@@ -167,6 +156,20 @@ pub fn run(
 
         let mut menu_selection = MenuSelection::Play;
         let mut volume = Volume::new();
+        let mut scene = Scene::new();
+        let tex = Rc::new(Texture::from_path("./images/rust-belt-logo.jpg").unwrap());
+        let mut sprite = Sprite::from_texture(tex.clone());
+        sprite.set_position(
+            window_size.width as f64 / 2.0,
+            (window_size.height as f64 / 2.0) - 120.0,
+        );
+        sprite.set_scale(0.4, 0.4);
+        let id = scene.add_child(sprite);
+        let fade = Sequence(vec![
+            Action(Ease(EaseFunction::QuadraticInOut, Box::new(FadeOut(0.0)))),
```

Comment:
Instead of having an instantaneous fade, we should probably set the opacity when we create it. I just now saw the setOpacity function in the API.